### ISSUE TITLE
CI CD pipeline is stuck on publishing package to JSR (Already uploaded, waiting for publishing)

### ISSUE THREAD

**Issue by kravetsone** [jsr-io/jsr#642]
[My repo with actions](https://github.com/kravetsone/t-kassa-api/actions/workflows/publish.yml)

![image](https://github.com/jsr-io/jsr/assets/57632712/0f81448c-0490-4492-af9b-0b043b32cd47)

[Action run with debug mode](https://github.com/kravetsone/t-kassa-api/actions/runs/9881630781/job/27314453578) 

my workflow file

```yml
name: Publish package

on: workflow_dispatch

permissions:
    contents: write
    id-token: write

jobs:
    publish_package:
        runs-on: ubuntu-latest

        steps:
            - name: Checkout
              uses: actions/checkout@v4
              with:
                  fetch-depth: 0

            - name: Install Bun
              uses: oven-sh/setup-bun@v1

            - id: changelog
              name: Generate changelog
              run: bun scripts/generate-changelog.ts

            - name: Install modules
              run: bun install

            - name: Build
              run: bunx pkgroll

            - name: Prepare to JSR publish
              run: bun scripts/release-jsr.ts

            - name: Type-check
              run: tsc --noEmit

            - name: Publish package to JSR
              run: bunx jsr publish --allow-dirty

            - name: Install Node
              uses: actions/setup-node@v4
              with:
                  node-version: "20.x"
                  registry-url: "https://registry.npmjs.org"

            - name: Publish package
              run: npm publish --access public
              env:
                  NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}

            - name: GitHub Release
              uses: ncipollo/release-action@v1
              with:
                  token: ${{ secrets.GITHUB_TOKEN }}
                  tag: v${{ steps.changelog.outputs.version }}
                  name: v${{ steps.changelog.outputs.version }}
                  body: ${{ steps.changelog.outputs.changelog }}
                  draft: false
                  prerelease: false
                  generateReleaseNotes: true
```

```bash
Resolving dependencies
Resolved, downloaded and extracted [16]
Saved lockfile
Downloading JSR binary...
Download completed
Checking for slow types in the public API...
Publishing @kravets/t-kassa-api@0.0.2 ...
Already uploaded, waiting for publishing @kravets/t-kassa-api@0.0.2
```

https://jsr.io/status/0e7e8436-65c9-478a-a547-ef25774dcb60

![image](https://github.com/jsr-io/jsr/assets/57632712/4b187373-58ca-48ad-9b9c-cbcafa09ca84)

**Comment by lucacasonato** [comment]
Sorry, we are working on it.

**Comment by ry** [comment]
Although publishing is going through, we're still working with GCP on the issue. We'll close this once completed.

**Comment by lucacasonato** [comment]
The incident is now resolved fully (but it should have been working already for the past ~6 hours again). The incident was caused by a faulty change within Google Cloud. I have asked them for an RCA report, that I will share here once I have it. Sorry for the troubles folks!

Also, I'm sorry that our internal monitoring did not catch this right away, and instead it was broken for so long before we noticed - the time to initial page on this one was totally unacceptable. Sorry! We're working now to fix this.
